services/gateway/apis/http-version2/methods/postTransactions.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const transactionsSource = require('../../../sources/version2/postTransactions');
+
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/transactions',
+	httpMethod: 'POST',
+	rpcMethod: 'post.transactions',
+	tags: ['Transactions'],
+	params: {
+		transaction: { optional: false, type: 'string', min: 1, pattern: /^\b[0-9a-fA-F]+\b$/ },
```

Comment:
Does the body JSON also count as a param? Or is it the URL param?

Reply:
Moleculer framework by default appends all the body JSON properties and request params into `$req.params`.
So, in our case `transaction` param is supposed to be part of the JSON object sent within the request body.